systemtest/src/main/java/io/strimzi/systemtest/resources/keycloak/SetupKeycloak.java

```diff
@@ -80,13 +87,36 @@ private static void deployKeycloak(ExtensionContext extensionContext, String nam
         LOGGER.info("Keycloak instance and Keycloak secret are ready");
     }
 
+    private static void deployPostgres(ExtensionContext extensionContext, String namespaceName) {
+        LOGGER.info("Deploying Postgres into namespace: {}", namespaceName);
+        cmdKubeClient(namespaceName).apply(POSTGRES_FILE_PATH);
+
+        DeploymentUtils.waitForDeploymentAndPodsReady(namespaceName, "postgres", 1);
+
+        ResourceManager.STORED_RESOURCES.get(extensionContext.getDisplayName()).push(new ResourceItem<>(() -> deletePostgres(namespaceName)));
+
+        Secret postgresSecret = new SecretBuilder()
+            .withNewMetadata()
+                .withName(POSTGRES_SECRET_NAME)
+                .withNamespace(namespaceName)
+            .endMetadata()
+                .withType("Opaque")
+                .addToData("username", Base64.getEncoder().encodeToString(POSTGRES_USER_NAME.getBytes(StandardCharsets.UTF_8)))
+                .addToData("password", Base64.getEncoder().encodeToString(POSTGRES_PASSWORD.getBytes(StandardCharsets.UTF_8)))
+            .build();
+
+
+        kubeClient().createSecret(postgresSecret);
+        SecretUtils.waitForSecretReady(namespaceName, "keycloak-db-secret", () -> { });
```

Comment:
Wouldn't be good to define `keycloak-db-secret` as a variable in the Constants class and reference it from there? In case that name will be used more than once...